CalibCalorimetry/EcalCorrectionModules/test/stubs/EcalContainmentCorrectionAnalyzer.cc

```diff
@@ -6,45 +6,44 @@
  *
  */
 
-#include <iostream>
-
-#include <FWCore/Framework/interface/EDAnalyzer.h>
+#include <FWCore/Framework/interface/one/EDAnalyzer.h>
 #include <FWCore/Framework/interface/Event.h>
 #include <FWCore/Framework/interface/MakerMacros.h>
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
 
 #include "CondFormats/DataRecord/interface/EcalGlobalShowerContainmentCorrectionsVsEtaRcd.h"
 #include "CondFormats/EcalCorrections/interface/EcalGlobalShowerContainmentCorrectionsVsEta.h"
-#include "FWCore/Framework/interface/ESHandle.h"
 #include "FWCore/Framework/interface/EventSetup.h"
 
 #include "DataFormats/EcalDetId/interface/EBDetId.h"
 
-class EcalContainmentCorrectionAnalyzer : public edm::EDAnalyzer {
+class EcalContainmentCorrectionAnalyzer : public edm::one::EDAnalyzer<> {
 public:
   EcalContainmentCorrectionAnalyzer(const edm::ParameterSet &ps);
   ~EcalContainmentCorrectionAnalyzer();
 
 protected:
   void analyze(edm::Event const &iEvent, const edm::EventSetup &iSetup);
```

Comment:
I suggest adding `override` designation here. Doing so is very helpful to guarantee that you never incorrectly type a signature of a base class virtual function.